tests/utils/esx/govc.go

```diff
@@ -54,11 +55,19 @@ func GetVMPowerState(vmName string) string {
 // PowerOnVM util powers on the VM
 // govc vm.power -on=true photon
 func PowerOnVM(vmName string) string {
-	// log.Printf("Powering on VM [%s]\n", vmName)
+	log.Printf("Powering on VM [%s]\n", vmName)
```

Comment:
Actually I don't really follow the comments at L#20. I think those one line log messages are pretty useful. Anyway, I will comment it out just for local consistency for now. Later we should re-enable it after introducing logging levels.